numpy/random/tests/test_random.py

```diff
@@ -799,6 +835,9 @@ def test_weibull(self):
                             [0.67057783752390987, 1.39494046635066793]])
         assert_array_almost_equal(actual, desired, decimal=15)
 
+    def test_weibull_0(self):
+        assert_equal(np.random.weibull(a=0), 0)
+
```

Comment:
Real nitpick, but for completeness might as well test for the exception being raise on `-0.` here as well.

This does look all OK otherwise.